api/agent/drivers/driver.go

```diff
@@ -86,6 +86,18 @@ type RunResult interface {
 	Status() string
 }
 
+// Logger Configuration for container
+type LoggerConfig struct {
+	// Log Sink URL
+	URL string
+
+	// Application Name for Log Tags
+	AppName string
+
+	// Function Name for Log Tags
+	FuncName string
```

Comment:
the driver package itself shouldn't have any knowledge of functions or apps, it only speaks in containers. could we change `AppName` and `FuncName` to `Tag string` as it's represented in the syslog config and pass in the string tag we'd like from the caller rather than configuring it from these variables? this is more flexible to future changes, too.

Reply:
i guess it's a little awkward with the `{{.Id}}` that the caller doesn't know about either. maybe the contract should just be to prepend whatever is provided ?

Reply:
upon further thought, should we be exposing the container id to the syslog logs? it makes sense maybe for us to be able to relate calls to a certain container for debugging stuff on the operational side, but do we want to expose to users this kind of information? most platforms seem to hide and google cloud functions even successfully masks it from users completely. for most users, they likely won't care at all though in the general case and the container id is just wasted bytes. it's also likely to spark questions (maybe with or without) around the nature of 'why did this function run here and this one there' (weak arg here)

Reply:
Maybe not {{.ID}}, but I think we need some kind of hot-container id, which is helpful to search in logs. We are expecting call-id to be added to these logs by FDKs, but call-id does not cover all (between calls, etc.). I could expose agent container id.

Reply:
in the hot container startup iirc we're logging to fn's stderr the container id + call id so that we can associate operationally, agree that this is useful. for user syslogs is this useful / should they care / should we hide it is kinda the open question

Reply:
I'm thinking like UDP syslog sink and interleaved logs. Maybe expose something like ctx_id and use agent container id for its value? Especially stdout/stderr with new lines. Or we can assume that FDK will tag everything (stdout+stderr) with a call_id?